### PR TITLE
DEV 2076 fix setuptools scm get version

### PR DESCRIPTION
**Pull request by qiaouchicago**

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -110,8 +98,8 @@ Docker build git tag:
   script:
     - BRANCH_NAME=$(git branch -a --contains tags/${CI_COMMIT_TAG} | grep origin | sed 's/.*origin\///' | tail -1)
     - CLEAN_BRANCH_NAME=${BRANCH_NAME/\//-}
-    - docker pull $DOCKER_RELEASE_REGISTRY/ncigdc/$CI_PROJECT_NAME:${CLEAN_BRANCH_NAME}
-    - docker tag $DOCKER_RELEASE_REGISTRY/ncigdc/$CI_PROJECT_NAME:${CLEAN_BRANCH_NAME} $DOCKER_RELEASE_REGISTRY/ncigdc/$CI_PROJECT_NAME:${CI_COMMIT_TAG}
+    - docker pull docker.osdc.io/ncigdc/$CI_PROJECT_NAME:${CLEAN_BRANCH_NAME}
```
Comment (wrkhenddher) [inline .gitlab-ci.yml:101]: bad merge?
Or are we hardcoding the server now?

File: dev-requirements.txt [hunk 2]
```diff
@@ -40,10 +40,11 @@ flask==2.0.3
     #   pytest-flask
 idna==3.4
     # via requests
-importlib-metadata==4.8.3
+importlib-metadata==4.8.3 ; python_version < "3.8"
```
Comment (wrkhenddher) [inline dev-requirements.txt:43]: still 3.7?

**Review by wrkhenddher — APPROVED** [review]

File: setup.py [hunk 3]
```diff
@@ -1,7 +1,18 @@
...
+this_directory = Path(__file__).parent
+long_description = (this_directory / "README.md").read_text()
+
 setup(
     name="indexd",
+    description="Data indexing and tracking service.",
+    author="NCI GDC",
+    author_email="gdc_dev_questions-aaaaae2lhsbell56tlvh3upgoq@cdis.slack.com",
+    url="https://github.com/NCI-GDC/indexd",
+    long_description=long_description,
```
Comment (kulgan) [inline setup.py:14]: 👍🏾

**Review by kulgan — APPROVED** [review]

File: dev-requirements.txt (hunk 2 above)
Comment (qiaouchicago) [reply dev-requirements.txt:43]: still 3.6~~~~~~

File: .gitlab-ci.yml (hunk 1 above)
Comment (qiaouchicago) [reply .gitlab-ci.yml:101]: Old one did not work, 
we have containers.osdc.io, dev-containers.osdc.io to push images for tag and branches, 
and could not get this work, @a-collet suggested to use this hard coded address.

**Pull request merged**